xbmc/Application.cpp

```diff
@@ -1081,10 +1081,30 @@ bool CApplication::InitDirectoriesLinux()
   else
     userHome = "/root";
 
+  /* Get XDG paths.
+   * Though a slash is expected at the end anyway, let's cover ourselves, just in case. */
+  std::string xdgDataPath;
+  if (getenv("XDG_DATA_HOME"))
+  {
+    xdgDataPath = getenv("XDG_DATA_HOME");
+    URIUtils::AddSlashAtEnd(xdgDataPath);
```

Comment:
URIUtils::AddSlashAtEnd() makes me feel sad. without it, for OpenELEc, I could
``` 
export XDG_DATA_HOME=/storage/.
```

but not now. 

switching from .xbmc to .kodi was enough pain.